plugin-tailwindcss/src/types.ts

```diff
@@ -1,48 +1,6 @@
-import type { OnTransformOptions } from "@fresh/core/dev";
-
-export interface AutoprefixerOptions {
-  /** environment for `Browserslist` */
-  env?: string;
-
-  /** should Autoprefixer use Visual Cascade, if CSS is uncompressed */
-  cascade?: boolean;
-
-  /** should Autoprefixer add prefixes. */
-  add?: boolean;
-
-  /** should Autoprefixer [remove outdated] prefixes */
-  remove?: boolean;
-
-  /** should Autoprefixer add prefixes for @supports parameters. */
-  supports?: boolean;
-
-  /** should Autoprefixer add prefixes for flexbox properties */
-  flexbox?: boolean | "no-2009";
-
-  /** should Autoprefixer add IE 10-11 prefixes for Grid Layout properties */
-  grid?: boolean | "autoplace" | "no-autoplace";
-
-  /** custom usage statistics for > 10% in my stats browsers query */
-  stats?: {
-    [browser: string]: {
-      [version: string]: number;
-    };
-  };
-
-  /**
-   * list of queries for target browsers.
-   * Try to not use it.
-   * The best practice is to use `.browserslistrc` config or `browserslist` key in `package.json`
-   * to share target browsers with Babel, ESLint and Stylelint
-   */
-  overrideBrowserslist?: string | string[];
-
-  /** do not raise error on unknown browser version in `Browserslist` config. */
-  ignoreUnknownVersions?: boolean;
-}
+import type { OnTransformOptions } from "fresh/dev";
 
 export interface TailwindPluginOptions {
```

Comment:
```suggestion
export interface TailwindPluginOptions extends PluginOptions {
```
From postcss

Reply:
I think a few things need to be addressed:
1. `TailwindPluginOptions` should be exported from `plugin-tailwindcss/src/mod.ts` (package entrypoint). Otherwise, that interface is inaccessible. IMO, the interface should be moved and `plugin-tailwindcss/src/types.ts` removed, but that might conflict with how Marvin wants things structured.
2. `PluginOptions` is defined in `@tailwindcss/postcss`, not `postcss`. But that's okay, we can just copy the `PluginOptions` interface properties, `base` (not sure if it's applicable for us) and `optimize`, to `TailwindPluginOptions`.
3. These interface properties should be documented. E.g. `optimize` defaults to `true` if Fresh is in production mode and `false` otherwise.